core/coreunix/add.go

```diff
@@ -316,6 +320,13 @@ func (adder *Adder) AddAllAndPin(file files.Node) (ipld.Node, error) {
 		return nil, err
 	}
 
+	if asyncDagService, ok := adder.dagService.(syncer); ok {
+		err = asyncDagService.Sync(nd.Cid())
```

Comment:
Syncing a _single_ block isn't going to do anything useful.

Reply:
Will change the interface here to not take CIDs, we'll end up syncing the entire blockstore